### PR TITLE
feat(Filters): add custom placeholder and units to input components

### PR DESCRIPTION
**Pull request by PrincAm**

### CODE REVIEW

File: src/components/Filters/FilterRow/FilterRow.tsx [hunk 1]
```diff
@@ -58,6 +58,15 @@ const SplitField = styled.div<SplitFieldProps>`
   }
 `;
 
+const Units = styled.span`
```
Comment (ajkl2533) [inline src/components/Filters/FilterRow/FilterRow.tsx:61]: why not `Text` component with `md` size?

File: src/components/Filters/components/DateRangePicker/DateRangePicker.tsx [hunk 2]
```diff
@@ -65,7 +66,7 @@ const DateRangePicker: React.FC<DateRangePickerProps> = ({
         endDate={endDate}
         maxDate={maxDate}
         minDate={startDate || minDate}
-        placeholderText="End date"
+        placeholderText={placeholder || 'End date'}
```
Comment (ajkl2533) [inline src/components/Filters/components/DateRangePicker/DateRangePicker.tsx:69]: Shouldn't we be able to pass separate placeholders for first and second field?

File: src/components/Filters/components/Input/Input.tsx [hunk 3]
```diff
@@ -52,6 +52,7 @@ const Input: React.FC<InputProps> = ({
   maxLength,
   pattern,
   patternMessage,
+  placeholder = 'String',
```
Comment (youssefssc) [inline src/components/Filters/components/Input/Input.tsx:55]: Another default placeholder than 'String' or an empty one would be better in this case

File: src/components/Filters/components/Number/Number.tsx [hunk 4]
```diff
@@ -24,7 +30,7 @@ const Number: React.FC<NumberProps> = ({ value = '', onChange, min, max }) => {
     <StyledNumber
       max={max}
       min={min}
-      placeholder="Number"
+      placeholder={placeholder}
       step="any"
```
Comment (ajkl2533) [inline src/components/Filters/components/Number/Number.tsx:34]: We probably should be able to set custom `step` value

File: src/components/Filters/components/Input/Input.tsx (hunk 3 above)
Comment (youssefssc) [reply src/components/Filters/components/Input/Input.tsx:55]: same with other placeholders: 'Date', 'Number', etc

File: src/components/Filters/components/DateRangePicker/DateRangePicker.tsx (hunk 2 above)
Comment (PrincAm) [reply src/components/Filters/components/DateRangePicker/DateRangePicker.tsx:69]: Originally was planned to have placeholder only for `Number` and I added it to all components. I was aware of this use case for start and end date and I din't want to add exception if it is not required. But it is not big deal

File: src/components/Filters/FilterRow/FilterRow.tsx (hunk 1 above)
Comment (PrincAm) [reply src/components/Filters/FilterRow/FilterRow.tsx:61]: Sure, I forget for `Text`

File: src/components/Filters/components/Input/Input.tsx (hunk 3 above)
Comment (PrincAm) [reply src/components/Filters/components/Input/Input.tsx:55]: Could you elaborate on that? `String`, `Date`, `Number` etc. are default placeholders defined by design

File: src/components/Filters/components/Number/Number.tsx (hunk 4 above)
Comment (PrincAm) [reply src/components/Filters/components/Number/Number.tsx:34]: We probably should but we also should display step buttons which are hidden now. It is design requirement. The change would be rather small. I will discuss it with UX and eventually get back to it under separate ticket.

File: src/components/Filters/components/Input/Input.tsx (hunk 3 above)
Comment (youssefssc) [reply src/components/Filters/components/Input/Input.tsx:55]: These were probably put there by Mate as labels not real practical placeholders

Comment (PrincAm) [reply src/components/Filters/components/Input/Input.tsx:55]: Those are real default placeholders. They're helping to end user to understand what data type can input and if there should be something more specific then developer can define custom placeholder which covers this PR :)

File: src/components/Filters/components/DateRangePicker/DateRangePicker.tsx (hunk 2 above)
Comment (PrincAm) [reply src/components/Filters/components/DateRangePicker/DateRangePicker.tsx]: Updated

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**